app/services/foreman_ansible/ansible_info.rb

```diff
@@ -14,5 +18,19 @@ def ansible_params
         memo
       end
     end
+
+    def ansible_roles
+      roles = host.all_ansible_roles.pluck(:id, :name)
```

Comment:
If `pluck` is removed, you will be able to use attribute names below instead of indexes.

Reply:
Wouldn't that be less efficient from a memory perspective? My thought was an array is more memory efficient than a hash and only half the fields are needed. I think the other fields are timestamps, but there could be more fields in the future.